src/js/permutive.js

```diff
@@ -109,6 +108,17 @@ class Permutive {
 		}
 		return Array.from(rootEl.querySelectorAll('[data-o-component="o-permutive"]'), rootEl => new Permutive(rootEl, opts));
 	}
+
+	/**
+	 * Add data to Permutive
+	 * @param {Object} userDemog
+	 * @param {Object} pageMeta
+	 */
+	static pAddon(userDemog, pageMeta) {
+		let user = { "user": Object.assign(userDemog) };
```

Comment:
I find these assign a bit hard to follow, do you think it could be clearer with a spread syntax?

Reply:
Yeah I agree, but I'll change that in the next PR to keep it simple

Reply:
@alexflorisca Are you trying to avoid mutating `userDemo`?. If so, I believe the correct syntax is:

```suggestion
		let user = { "user": Object.assign({}, userDemog) };
```

Reply:
Unless the code gets transpiled by OBT - or we use babel -, it might not be the best idea to use the spread object instead of Object.assign, as some browser might still not support it.

I think that the first Object.assign could be simply removed
```
{ user: Object.assign(userDemog) }
```
is the same as
```
{user: userDemog}
```
, while the second Object.assign would better be rewritten adding an aempty object as first argument, as follow.
```
let data = {page: Object.assign({}, pageMeta, user)}
```
It's just good practice, in order to make sure that pageMeta is not overwritten with attributes coming from the object user